lib/modules/datasource/metadata-manual.ts

```diff
@@ -46,14 +46,27 @@ export const manualChangelogUrls: Record<string, Record<string, string>> = {
     wagtail: 'https://github.com/wagtail/wagtail/tree/master/docs/releases',
   },
   docker: {
+    'confluentinc/ksqldb-cli': 'https://github.com/confluentinc/ksql',
```

Comment:
those all should be in manuals source urls below instead.

Reply:
thanks @viceice, resolved in  a073acf